### ISSUE TITLE
Thanks for implementing --csv-stats. It works a little differently than I expected. Is it possible to correct this.

### ISSUE THREAD

**Issue by trust1345** [HynekPetrak/log4shell-finder#18]
Thanks for implementing --csv-stats. It works a little differently than I expected. Is it possible to correct this.
I liked the proposed implementation.
1 separate row for each host, instead of adding statistics to each row by adding extra columns (when the --csv-stats argument is specified). This approach is better than I originally suggested.

launched on https://github.com/HynekPetrak/log4shell-finder/commit/75cb4f129110f5da8e8132698ed2b52ad34c405c
`test_log4shell_1.2120220126.py all --csv-out --no-csv-header --no-error --csv-clean --csv-stats`
[output2.csv](https://github.com/HynekPetrak/log4shell-finder/files/7940889/output2.csv)
[output1.csv](https://github.com/HynekPetrak/log4shell-finder/files/7940895/output1.csv)

Proposed format
[output1-expected.csv](https://github.com/HynekPetrak/log4shell-finder/files/7940900/output1-expected.csv)
[output2-expected.csv](https://github.com/HynekPetrak/log4shell-finder/files/7940901/output2-expected.csv)
Also in the proposed output, I propose to slightly change the output of --csv-clean, for unification

thank you in advance.

**Comment by HynekPetrak** [comment]
STATS line is printed for each folder / drive given on the command line. So if you have c:\ d:\ there will be two STATS lines and therefore I used the path column for a directory name.
Let me know, how you would see the STATS line with the directory/drive added.

**Comment by trust1345** [comment]
A separate STATS line for each folder/drive does not seem very convenient to me. It seems to me it will be more convenient if you do as in the example:
[output1-expected.csv](https://github.com/HynekPetrak/log4shell-finder/files/7940900/output1-expected.csv)

One row of STATS for all folders/drives (as well as the total time and number of files and folders), and in the path column simply indicate the full line of the startup.
![Снимок экрана от 2022-01-31 00-33-28](https://user-images.githubusercontent.com/64325530/151718818-5afaaa7b-1be4-49e2-af03-c9a08251ace0.png)

**Comment by HynekPetrak** [comment]
@trust1345 corrected as you suggested. And sorry for delay, I've been busy.
I have combined the two lines `STATS` with `CLEAN` status as they contain the same information anyway.

```csv
"2022-02-22 22:09:03","1.22pre-20220128","10.154.24.65","hostname","Linux","5.4.0-58-generic","x86_64","","CLEAN, STATS","./test_log4shell.py ./__pycache__/ --same-fs -c -j --csv-stats --csv-clean --no-csv-header","0.014085054397583008","1","1"
```

**Comment by HynekPetrak** [comment]
corrected in 450de69
